api/src/main/resources/application.properties

```diff
@@ -28,3 +28,13 @@ spring.jpa.properties.hibernate.dialect=org.pmiops.workbench.cdr.MySQLDialect
 # and https://precisionmedicineinitiative.atlassian.net/browse/RW-1478
 # Disable all spring-boot endpoints unless specifically necessary
 management.endpoints.enabled-by-default=false
+
+# DB Connection set by environment variables
+spring.datasource.driver-class-name=${DB_DRIVER:com.mysql.jdbc.Driver}
+spring.datasource.url=${DB_CONNECTION_STRING:jdbc:mysql://127.0.0.1/workbench?useSSL=false}
+spring.datasource.username=${WORKBENCH_DB_USER:workbench}
+spring.datasource.password=${WORKBENCH_DB_PASSWORD:wb-notasecret}
+cdr.datasource.url=${CDR_DB_CONNECTION_STRING:jdbc:mysql://127.0.0.1/workbench?useSSL=false}
```

Comment:
it looks like these values are duplicated from `api/db/vars.env`. Is it possible to load this from there?

Reply:
I made those default variable for your use case, i.e. `./gradlew appengineRu`n. Now it should be `./gradlew bootRun`, then this file becomes the entry point, but seems it still not working. I may remove those default value, and put everything inside the ruby or vars.env...

Reply:
ah, got it. thanks. you don't have to worry about my set up for now. I'll resolve that after you merge this in.